### PR TITLE
Accept user input

### PR DESCRIPTION
**Pull request by maranhaoca**
Improvement to accept user input

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -21,3 +21,4 @@
 
 # virtual machine crash logs, see http://www.java.com/en/download/help/error_hotspot.xml
 hs_err_pid*
+.idea/*
```
Comment (bragalivia) [inline .gitignore]: Please, add a new emptu line at the end of the file

File: src/UniqueCharacters.java [hunk 2]
```diff
@@ -1,10 +1,19 @@
+import java.util.Scanner;
+
 public class UniqueCharacters {
     public static void main(String[] args) {
-        System.out.println(uniqueCharacters("caio"));
+        Scanner input = new Scanner(System.in);
+        System.out.println("Welcome to UniqueCharacters!!\nEnter a string!!");
```
Comment (bragalivia) [inline src/UniqueCharacters.java:5]: put the /welcome message at the first line of the main method

File: src/UniqueCharacters.java [hunk 3]
```diff
@@ -1,10 +1,19 @@
...
+        String userInput = input.next();
+        if (uniqueCharacters(userInput)) {
+            System.out.println("It's " + uniqueCharacters(userInput) + ", this string has unique characters.");
```
Comment (bragalivia) [inline src/UniqueCharacters.java]: to avoid call multiple times uniqueCharacters method, please get the return and use a local variable

**Review by bragalivia — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 1 above)
Comment (maranhaoca) [reply .gitignore]: Done.

File: src/UniqueCharacters.java (hunk 2 above)
Comment (maranhaoca) [reply src/UniqueCharacters.java:5]: Done.

File: src/UniqueCharacters.java (hunk 3 above)
Comment (maranhaoca) [reply src/UniqueCharacters.java]: Done.

**Review by bragalivia — APPROVED** [review]

**Pull request merged**